content/general/concepts/middleware/middleware.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Middleware'
+Description: 'Middleware is software that connects applications together. Middleware connects data, APIs, software tools and other applications in other to bring a complete application to the end user.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
+  - 'APIs'
+  - 'Database'
+  - 'Middleware'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-html'
+  - 'paths/back-end-engineer-career-path'
+---
+
+Middleware is software that connects applications together. The term _middleware_ comes from the fact that this software sits between the front-end or client side of an application and the back-end where the resources are being requested. Middleware connects or "glues" together data, APIs, software tools and other applications in order to bring a complete or bundled application to the end user. There are many types of middleware software depending on the use cases. Some of the most commonly used middleware is listed below.
+
+## Types of Middleware
+
+### Message-oriented middleware
+
+This type of middleware allows applications to exchange messages and communicate between eachother.
```

Comment:
```suggestion
A message oriented middleware, or MOM, allows applications to exchange messages and communicate between one another.
```

Could include a short list of prominent examples for each of these middleware types?